ord_schema/units.py

```diff
@@ -138,6 +138,63 @@
     "m": "ambiguous between meter and minute",
 }
 
+_UNIT_CONVERSIONS = {
+    reaction_pb2.Time: {
+        reaction_pb2.Time.DAY: 24,
+        reaction_pb2.Time.HOUR: 1,
+        reaction_pb2.Time.MINUTE: 1 / 60,
+        reaction_pb2.Time.SECOND: 1 / 3600,
+    },
+    reaction_pb2.Mass: {
+        reaction_pb2.Mass.GRAM: 1000,
+        reaction_pb2.Mass.MILLIGRAM: 1,
+        reaction_pb2.Mass.MICROGRAM: 0.001,
+        reaction_pb2.Mass.KILOGRAM: 1000000,
+    },
+    reaction_pb2.Moles: {
+        reaction_pb2.Moles.MOLE: 1000,
+        reaction_pb2.Moles.MILLIMOLE: 1,
+        reaction_pb2.Moles.MICROMOLE: 0.001,
+        reaction_pb2.Moles.NANOMOLE: 0.000001,
+    },
+    reaction_pb2.Volume: {
+        reaction_pb2.Volume.MILLILITER: 1,
+        reaction_pb2.Volume.MICROLITER: 0.001,
+        reaction_pb2.Volume.LITER: 1000,
+        reaction_pb2.Volume.NANOLITER: 0.000001,
+    },
+    reaction_pb2.Length: {
+        reaction_pb2.Length.CENTIMETER: 1,
+        reaction_pb2.Length.MILLIMETER: 0.1,
+        reaction_pb2.Length.METER: 100,
+        reaction_pb2.Length.INCH: 2.54,
+        reaction_pb2.Length.FOOT: 30.48,
+    },
+    reaction_pb2.Pressure: {
+        reaction_pb2.Pressure.BAR: 1 / 1.01325,
+        reaction_pb2.Pressure.ATMOSPHERE: 1,
+        reaction_pb2.Pressure.PSI: 1 / 14.69595,
+        reaction_pb2.Pressure.KPSI: 1000 / 14.69595,
+        reaction_pb2.Pressure.PASCAL: 1 / 101325,
+        reaction_pb2.Pressure.KILOPASCAL: 1000 / 101325,
+    },
+    reaction_pb2.Current: {
+        reaction_pb2.Current.AMPERE: 1,
+        reaction_pb2.Current.MILLIAMPERE: 0.001,
+    },
+    reaction_pb2.Voltage: {
+        reaction_pb2.Voltage.VOLT: 1,
+        reaction_pb2.Voltage.MILLIVOLT: 0.001,
+    },
+    reaction_pb2.FlowRate: {
```

Comment:
Am I crazy or `1 uL/min` should be `1/60 uL/s`?

Reply:
My strategy here was to peg everything to a reference of 1 'unit'. So for example, one `VOLT` is 1 unit (V), one `MILLIVOLT` is 0.001 unit (V).

One `MICROLITER_PER_SECOND` is 60x as high of a flowrate as one `MICROLITER_PER_MINUTE`